code/src/App.js

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom';
+import PopularList from 'components/PopularList';
+import Details from 'components/Details';
+import NotFound from 'components/NotFound';
 
 export const App = () => {
+  const [popularList, setPopularList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  console.log(loading)
+
+  useEffect(() => {
+    setLoading(true);
+    fetch('https://api.themoviedb.org/3/movie/popular?api_key=22332a0afa34a5b14d4e80b45a24b1e8&language=en-US&page=1')
+      .then((res) => res.json())
+      .then((data) => {
+        setPopularList(data.results)
+      })
+      .catch((e) => {
+        console.error(console.error(e))
+      })
+      .finally(() => {
+        setLoading(false);
+      })
+  }, []);
+
```

Comment:
Interesting that you have put the fetch here in app.js, now we have learned that you don't have to put it in the same component as the . map. :)

Reply:
Thanks for the feedback, we probably lost some information here since we tried to follow the example-project. We'd be happy for a hint on where you got other information. :)